### DISCUSSION TITLE
Обратное переключение из multi в nomulti

### DISCUSSION THREAD

**Discussion by LostAlly** [totumonline/totum-ru-issues-and-discussions#364] [Как это сделать?] [answered]
Приветствую, 
Можно ли из мульти переключиться в номульти обратными действиями?
В документации не увидел запрета или, что можно обратными действиями.
Спасибо.

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Да можно - надо измеить Conf.php и переместить файлы.

Но это какой-то специальный случай, например у вас доступ к тотум по ip и он динамический.

**Reply by ttm-alexeypolunin** [reply]
Те мы разработали изначально именно multi. No-multi был сделан для установки на шаред netangels.

**Reply by LostAlly** [reply]
Спасибо, тогда не буду дергать, просто я почему то решил сначала, что мульти нужно для разработки например на одном домене идет разработка, а на другом рабочий. А потом я решил, что можно делать в nomulti и у него есть сложности(я пока разбираюсь, поэтому мое понимание тотума смутно)))
